### PR TITLE
Implement gitignore for Plugins

### PR DESCRIPTION
**Pull request by jcampbell05**
Potentially we should also add a method or step to initialize a git repo.

### CODE REVIEW

File: fastlane/lib/fastlane/plugins/template/.gitignore [hunk 1]
```diff
@@ -1,10 +1,39 @@
...
+*.rbc
+/.config
+/coverage/
+/InstalledFiles
+/pkg/
+/spec/reports/
+/test/tmp/
+/test/version_tmp/
+/tmp/
+fastlane-plugin-xcake/Gemfile.lock
```
Comment (KrauseFx) [inline fastlane/lib/fastlane/plugins/template/.gitignore]: Should this be here?

**Comment by KrauseFx** [comment]
Shouldn't the `.gitignore` be taken from the fastlane repo is some way, so that we don't have to maintain 2 files?

File: fastlane/lib/fastlane/plugins/template/.gitignore (hunk 1 above)
Comment (jcampbell05) [inline fastlane/lib/fastlane/plugins/template/.gitignore]: Nope it should not

**Comment by jcampbell05** [comment]
@KrauseFx I guess so, I guess add that to the scaffolding script? Know where in the code it should be added ?

**Comment by KrauseFx** [comment]
This should be copied from the fastlane `.gitignore`, so we don't have to have duplicates 👍

**Pull request closed without merging**